.semaphore/deploy_npmjs.yml

```diff
@@ -16,4 +16,4 @@ blocks:
 agent:
   machine:
     type: e1-standard-2
-    os_image: ubuntu2004
+    os_image: ubuntu2204
```

Comment:
Thanks, I will take into account for my next commit!

Reply:
@koral-- , I have a question about the `Dockerfile` file because I would like to optimize it, why the `bundletool` package is installed because it is unused and not useful isn't it?